corehq/apps/hqadmin/views.py

```diff
@@ -643,7 +650,7 @@ def _lookup_id_in_database(doc_id, db_name=None):
         else:
             db_results.append(db_result(db.dbname, 'found', 'success'))
             response.update({
-                "doc": json.dumps(doc, indent=4, sort_keys=True),
+                "doc": json.dumps(doc, indent=4, sort_keys=True, cls=DjangoJSONEncoder),
```

Comment:
`DjangoJSONEncoder` will convert `datetime` objects instead of throwing an exception.